apps/condo/pages/auth/change-password.tsx

```diff
@@ -154,7 +186,7 @@ const ChangePasswordPage: AuthPage = () => {
                         <Button
                             key='submit'
                             type='sberPrimary'
-                            loading={isLoading}
+                            loading={isSaving}
```

Comment:
Maybe, better to use loading as it has conflict in naming here

Reply:
@dkoviazin 
Loading in my case related to whole page, ad saving for 'Save' button indicator